spec/support/capybara_actions.rb

```diff
@@ -9,6 +9,16 @@ def blur_from(locator)
   def within_section(title, &)
     within(:xpath, "//section[contains(header/text(),'#{title}')]", &)
   end
+
+  # An alias so our tests are less coupled to the aXe implementation
+  def be_accessible(...)
+    be_axe_clean(...).according_to(
+      :wcag2a,
+      :wcag2aa,
+      :"best-practice",
+      :experimental
```

Comment:
@lwrubel Here you go.

Reply:
See also https://github.com/dequelabs/axe-core/blob/master/doc/rule-descriptions.md for a list of the rules and which tags (e.g., `:wcag2aa`) include them.